modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -62,6 +62,7 @@ class GrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   virtual std::size_t getNumberActiveGrains() const override;
   virtual Point getGrainCentroid(unsigned int grain_id) const override;
   virtual bool doesFeatureIntersectBoundary(unsigned int feature_id) const override;
+  virtual std::vector<unsigned int> getNewGrainIDs() const override;
```

Comment:
Is returning the vector by value (rather than filling in a vector passed in by reference) the most "MOOSE-like" way of doing this?  Not saying that efficiency would even be a concern for this particular function, just asking a question.